### ISSUE TITLE
Device independent control should be consistent

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1078]
Device independent control on the AAPB website should be consistent. Currently, some focus is a default dotted box that is not very visible. Some focus is a more defined blue box.
